plugins.json

```diff
@@ -526,7 +618,7 @@
       },
       "settings_schema": {
         "header": "",
-        "footer": "Use this webhook URL to set up the Jira integration. See [documentation](https://about.mattermost.com/default-jira-plugin) to learn more.\n\n`https://SITEURL/plugins/jira/webhook?secret=WEBHOOKSECRET&team=TEAMURL&channel=CHANNELURL`",
+        "footer": "Use this webhook URL to set up the Jira integration. See [documentation](https://about.mattermost.com/default-jira-plugin) to learn more.\n\n`https://SITEURL/plugins/jira/webhook?secret=WEBHOOKSECRET\u0026team=TEAMURL\u0026channel=CHANNELURL`",
```

Comment:
@iomodo is this unicode `\u0026` encoding intentional? Is `jq` doing this?

Reply:
generator generated this way, not sure why

Reply:
Yeah, this worries me slightly too. It technically won't cause a problem, but good to get to the bottom of why.

Reply:
Agreed. I normally do `go run ./cmd/generator --github-token $GITHUB_TOKEN --debug | jq > plugins.json` and don't notice the encoding locally 🤔